toqito/rand/random_psd_operator.py

```diff
@@ -0,0 +1,59 @@
+"Generates a random positive semi-definite operator."
+
+import numpy as np
+
+def random_psd_operator(dim: int) -> np.ndarray:
+    r''' Generate a random positive semi definite operator.
+
+    A positive semi-definite operator is a Hermitian operator that has only real and non-negative eigenvalues. 
+
+    This function generates a random positive semi-definite operator by constructing a symmetric matrix, 
```

Comment:
This docstring needs to be written more clearly. For instance "eigenvalues" (not "eigen values"), statements like "The resultant of the function ..." could be worded more clearly, "creates a symmetry matrix" (it's not a "symmetry matrix" it's a "symmetric matrix", etc.